nilearn/decomposition/multi_pca.py

```diff
@@ -176,30 +103,36 @@ class MultiPCA(BaseEstimator, TransformerMixin, CacheMixin):
         the `masker_` attribute.
     """
 
-    def __init__(self, n_components=20, smoothing_fwhm=None, mask=None,
-                 do_cca=True, standardize=True, target_affine=None,
-                 target_shape=None, low_pass=None, high_pass=None,
-                 t_r=None, random_state=None,
+    def __init__(self, n_components=20, do_cca=True,
```

Comment:
IMHO it is better : we put all estimator related argument first, then masker related argument, then system related argument